package.json

```diff
@@ -41,7 +41,9 @@
     "helmet": "^3.21.1",
     "morgan": "^1.9.1",
     "ngx-webstorage-service": "^4.1.0",
-    "rxjs": "~6.5.0",
+    "puppeteer-core": "^1.20.0",
+    "rxjs": "^6.5.3",
+    "rxjs-compat": "^6.5.3",
```

Comment:
@tbg-jasonfung We shouldn't be adding in `rxjs-compat` to a new project - this lib is for compatibility with the old API, so the right approach is to find the new versions of the operators.